x/gov/codec.proto

```diff
@@ -89,7 +92,10 @@ message Proposal {
   // Reference to the election rule
   bytes election_rule_id = 4 [(gogoproto.customname) = "ElectionRuleID"];
   // Reference to the electorate to define the group of possible voters.
-  bytes electorate_id = 5 [(gogoproto.customname) = "ElectorateID"];
+  orm.VersionedIDRef electorate_ref = 5 [
+    (gogoproto.customname) = "ElectorateRef",
```

Comment:
Isn't this also the default name. I mean, avoid customname directive unless needed